lib/views/Graph.js

```diff
@@ -30,7 +31,7 @@ class Graph extends GuideLayer {
         return 0;
     }
 
-    drawElement(d3Svg, xRange, yRange, axisLabels) {
+    drawElement(d3Svg, xRange, yRange, axisLabels, dontAnimateOverride = false) {
```

Comment:
Took me a few reads to get this. How about `dontAnimateOverride` --> `dontAnimate`? 

Or maybe it would be clearer to have the argument be `animate = true`. A little bit more logic needed, but I think that might scan a little better.